content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -9,6 +9,22 @@ menu:
 v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ---
 
+## v1.8.4 [unreleased]
+
+### Features
+
+- Optimize shard lookups in groups containing only one shard. Thanks @StoneYunZhao!
+- Add [`strict-error-handling` configuration option](/influxdb/v1.8/administration/config/#strict-error-handling-false). Previously, SELECT INTO queries attempting to insert an unsupported value failed silently. To preserve backwards compatibility, a new `strict-error-handling` option was added to the InfluxDB configuration file. Set this option to `true` to ensure a failed insert returns an error and no points are inserted.
+
+### Bug fixes
+
+- Ensure successful writes increment `WriteOK` write statistics correctly.
+- Add durations to Flux logging, including log compilation, execution, and total request duration.
+- Properly format JSON marshalling errors.
+- Close all `net.Listener` instances silently on error.
+- `regexp` handling complies with Prometheus Query Language (PromQL).
```

Comment:
add "in the prometheus converter" to make it clear that this doesn't affect regexp handling universally.

Reply:
thank you, @davidby-influx!